doc/source/operations/upgrading.rst

```diff
@@ -149,6 +149,16 @@ Known issues
 
      curl -X PUT http://<controller-ip>:9200/_cluster/settings -H 'Content-Type:application/json' -d '{"transient":{"cluster":{"routing":{"allocation":{"enable":"all"}}}}}'
 
+* Docker log-opts are currently not configured in Antelope. You will see these
+  being removed when running a host configure in check+diff mode. See bug for
+  details (fix in progress):
+  https://bugs.launchpad.net/ansible-collection-kolla/+bug/2040105
+
+* /etc/hosts are not templated correctly when running a host configure with
+  ``--limit``. If you don't need to change /etc/hosts, you can work around this
+  with ``--skip-tags etc-hosts``. See bug for details (fix in progress):
+  https://bugs.launchpad.net/kayobe/+bug/2051714
```

Comment:
and one can also run the ```kayobe overcloud host configure -t etc-hosts``` on all cloud after adding new hosts and running host configure with limit for example

Reply:
That's a good point, I've updated to include this.